services/s3.cfc

```diff
@@ -290,6 +290,172 @@ component {
         return apiResponse;
     }
 
+
+    /**
+     * This action initiates a multipart upload and returns an upload ID. This upload ID is used to associate all of the parts in the specific multipart upload. You specify this upload ID in each of your subsequent upload part requests (see UploadPart). You also include this upload ID in the final request to either complete or abort the multipart upload request.
+     * https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateMultipartUpload.html
+     * 
+     *
+     * @Bucket: The name of the bucket to which to initiate the upload
+     * @Key: Object key for which the multipart upload is to be initiated.
+     */
+    public any function createMultipartUpload(
+        required string Bucket,
+        required string Key
+    ){
+
+         var requestSettings = api.resolveRequestSettings( argumentCollection = arguments );
+
+        var queryParams = { 'uploads': '' };
+        for (
+            var key in arguments
+        ) {
+            // if ( len( arguments[ key ] ) ) queryParams[ utils.parseKey( key ) ] = arguments[ key ];
```

Comment:
I take it nothing from the arguments needs to go into the query string? Should this be removed then?